flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/config/ExecutionConfigOptions.java

```diff
@@ -121,6 +121,27 @@
                                     + "into NOT NULL columns. Users can change the behavior to 'drop' to "
                                     + "silently drop such records without throwing exception.");
 
+    @Documentation.TableOption(execMode = Documentation.ExecMode.BATCH_STREAMING)
+    public static final ConfigOption<CharPrecisionEnforcer>
+            TABLE_EXEC_SINK_CHAR_PRECISION_ENFORCER =
+                    key("table.exec.sink.char-precision-enforcer")
+                            .enumType(CharPrecisionEnforcer.class)
+                            .defaultValue(CharPrecisionEnforcer.ERROR.ERROR)
+                            .withDescription(
```

Comment:
Please don't list enum values, their explanation, or the default value in the description; rather, document the enum values (implementing `DescribedEnum`). The docs generator van then do a better job.